app/uk/gov/hmrc/fileupload/microserviceGlobal.scala

```diff
@@ -17,83 +17,121 @@
 import uk.gov.hmrc.play.audit.filters.AuditFilter
+import uk.gov.hmrc.play.audit.http.config.ErrorAuditingSettings
+import uk.gov.hmrc.play.audit.http.connector.AuditConnector
 import uk.gov.hmrc.play.auth.controllers.AuthParamsControllerConfig
 import uk.gov.hmrc.play.auth.microservice.filters.AuthorisationFilter
-import uk.gov.hmrc.play.config.{AppName, ControllerConfig, RunMode}
-import uk.gov.hmrc.play.http.BadRequestException
+import uk.gov.hmrc.play.config.{AppName, ControllerConfig}
+import uk.gov.hmrc.play.graphite.{GraphiteConfig, GraphiteMetricsImpl}
 import uk.gov.hmrc.play.http.logging.filters.LoggingFilter
-import uk.gov.hmrc.play.microservice.bootstrap.DefaultMicroserviceGlobal
+import uk.gov.hmrc.play.microservice.bootstrap.Routing.RemovingOfTrailingSlashes
+import uk.gov.hmrc.play.microservice.bootstrap.{JsonErrorHandling, MicroserviceFilters}
 
-import scala.concurrent.{ExecutionContext, Future}
 
-object ControllerConfiguration extends ControllerConfig {
-  lazy val controllerConfigs = Play.current.configuration.underlying.as[Config]("controllers")
-}
-
-object AuthParamsControllerConfiguration extends AuthParamsControllerConfig {
-  lazy val controllerConfigs = ControllerConfiguration.controllerConfigs
-}
-
-object MicroserviceAuditFilter extends AuditFilter with AppName {
-  override val auditConnector = MicroserviceAuditConnector
-
-  override def controllerNeedsAuditing(controllerName: String) = ControllerConfiguration.paramsForController(controllerName).needsAuditing
+class ApplicationLoader extends play.api.ApplicationLoader {
+  def load(context: Context) = {
+    LoggerConfigurator(context.environment.classLoader).foreach {
+      _.configure(context.environment)
+    }
+    new ApplicationModule(context).application
+  }
 }
 
-object MicroserviceLoggingFilter extends LoggingFilter {
-  override def controllerNeedsLogging(controllerName: String) = ControllerConfiguration.paramsForController(controllerName).needsLogging
-}
+class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(context)
+  with AhcWSComponents with AppName with MicroserviceFilters
+  with GraphiteConfig
+  with RemovingOfTrailingSlashes
+  with JsonErrorHandling
+  with ErrorAuditingSettings {
 
-object MicroserviceAuthFilter extends AuthorisationFilter {
-  override lazy val authParamsConfig = AuthParamsControllerConfiguration
-  override lazy val authConnector = MicroserviceAuthConnector
+  var subscribe: (ActorRef, Class[_]) => Boolean = _
+  var publish: (AnyRef) => Unit = _
+  var withBasicAuth: BasicAuth = _
+  var eventStore: MongoEventStore = _
 
-  override def controllerNeedsAuth(controllerName: String): Boolean = ControllerConfiguration.paramsForController(controllerName).needsAuth
-}
+  implicit val reader = new UnitOfWorkReader(EventSerializer.toEventData)
+  implicit val writer = new UnitOfWorkWriter(EventSerializer.fromEventData)
 
-object MicroserviceGlobal extends DefaultMicroserviceGlobal with RunMode {
```

Comment:
Perhaps the name of the file should change given we no longer have a Global Settings in here